### PR TITLE
Create Video Looper.md

### PR DESCRIPTION
**Pull request by fix-ON**

### CODE REVIEW

File: Raspberry Pi/Video Looper.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Video Looper
+This is a simplified guide for how to install the **Adafruits - Rpi Video Looper**. This is an application that turns your Raspberry Pi into a dedicated looping video playback device. This can be used in art installations, fairs, theatre, events, infoscreens, advertisment etc...
```
Comment (tevey) [inline Raspberry Pi/Video Looper.md:2]: I think "infoscreens" should be two words "info screens"

Comment (tevey) [inline Raspberry Pi/Video Looper.md:2]: advertisment should be advertisement

File: Raspberry Pi/Video Looper.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+When using the Raspberry Pi ZERO, you might want to use the Analog audio only. Using both HDMI and Analog audio out will cause stuttering because the Rpi ZERO is not powerful enough. You can change this in the `video_looper.ini` which is explained above:
+
+       #sound = both
+       #sound = hdmi
+       sound = local
+
+## To-do
+
+- Add instructions for removing/disable the Video Looper.
+
```
Comment (tevey) [inline Raspberry Pi/Video Looper.md:71]: remove unnecessary whitespace at end of file

**Review by tevey — CHANGES_REQUESTED** [review]

**Review by fix-ON — COMMENTED** [review]
Fixed

**Review by tevey — APPROVED** [review]

**Pull request merged**